arrow-core-data/src/main/kotlin/arrow/core/Either.kt

```diff
@@ -901,6 +900,21 @@ sealed class Either<out A, out B> : EitherOf<A, B> {
       } catch (t: Throwable) {
         fe(t.nonFatalOrThrow()).left()
       }
+
+    fun <E, A> fx2(c: suspend EagerBind<EitherPartialOf<E>>.() -> A): Either<E, A> {
```

Comment:
`fxEager`? `fxNow`? `fxJust`?

Reply:
I'd like to merge as soon as the build is green to continue working with other tickets, so I created an issue to discuss the naming in order not to block us :) Please feel free to continue the discussion there: https://github.com/arrow-kt/arrow-core/issues/133